src/main/webapp/app/course/manage/course-management.service.ts

```
@@ -703,4 +711,13 @@
     disableCourseOverviewBackground() {
         this.courseOverviewSubject.next(false);
     }
+
+    getSemesterCollapseStateFromStorage(storageId: string): boolean {
+        const storedCollapseState: string | null = localStorage.getItem('semester.collapseState.' + storageId);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)